src/lib/setup.ts

```diff
@@ -0,0 +1,21 @@
+import ddTracer from "dd-trace"
+
+export const setupDataDog = () => {
+  if (process.env.DD_APM_ENABLED) {
```

Comment:
we have to create ENV variables (`DD_APM_ENABLED` and `DD_TRACE_AGENT_HOSTNAME`) both on staging and production before merging this one?

Reply:
@xtina-starr we should also add them to the `.env.example` file

Reply:
Does `DD_TRACE_AGENT_HOSTNAME` not get picked up from hokusai [here](https://github.com/artsy/kaws/blob/452936f09b54b351905a5cbde65d7948c36f3b98/hokusai/staging.yml#L24)?

Reply:
My bad, I missed that, it should actually. I'll merge and see if we get any issues